aws/README.md

```diff
@@ -0,0 +1,38 @@
+## Launching a Convox Rack on AWS and deploying an application
+
+- Install the latest Convox CLI for your operating system using the instructions found here: https://docs.convox.com/introduction/installation
+- Run `convox update`
+- Create a [convox.yml](https://docs.convox.com/application/convox-yml) file to describe your application
+- Sign up for AWS if you have not already (https://signin.aws.amazon.com/)
+- Ensure that your AWS credentials are set on your local machine.  We like using [aws-vault](https://github.com/99designs/aws-vault) but having your Access Key ID and Secret Access Key set as env vars should do fine as well:
+    ```
+    $ export AWS_ACCESS_KEY_ID="anaccesskey"
+    $ export AWS_SECRET_ACCESS_KEY="asecretkey"
+    $ export AWS_DEFAULT_REGION="us-west-2"
+    ```
+- If you don't already have these values, you can obtain them from the IAM service dashboard within AWS.
+- Clone https://github.com/convox-examples/terraform into a local directory and change to that directory
+- Go into the `aws` subdirectory
+- Run `terraform init`
+- Run `terraform apply`
+- When prompted enter the name of a domain that you have DNS control over
+  - for example: `convox.mydomain.com`
+- Follow the terraform prompts 
+- When `terraform apply` completes you should see an output named `rack_url`. Set a local environment variable named `RACK_URL` to that value in your shell
+  - `$ export RACK_URL=https://convox:ebXYv6904JMpA7ArnFhw9Ml1iJtOa3cZhP4pWyVrOShl4xZpBvADeOqr7aByxQ1R@api.convox.mydomain.com=`
```

Comment:
I would shorten this so it doesn't wrap on most screens. I think they will get the point